### PR TITLE
finished homework06

### PR DESCRIPTION
**Pull request by zoyazip**

### CODE REVIEW

File: Majasdarbs/06-majasdarbs/homework/script.js [hunk 1]
```diff
@@ -1,39 +1,49 @@
...
-        izmaiņam defineto output mainigo iekšejo text (innerText) ar izveleto burtu masiva (characters[arrayId])
-*/
-
-// 6) izsaucam funkciju getValue()
-
-/* 7) japievino event listener keyPress un parbaudit ievadito to tastaturas vertibu ar izveleto vertibu pirm tam
-        ja gadijuma izvadit paziņojumu alert('PAREIZI') un izsaukt funkciju getValue() vel reizi
-        ne gadijum izvadit paziņojumu alert('NEPAREIZI')
-*/      
+const letters = [
```
Comment (zoyazip) [inline Majasdarbs/06-majasdarbs/homework/script.js:1]: do it in 1 line if its possible

**Review by zoyazip — COMMENTED** [review]
iesk, but u alse could try to add a message, if promt is empty and if promt was canceled(undefined in your case).

File: Majasdarbs/06-majasdarbs/homework/script.js [hunk 2]
```diff
@@ -1,39 +1,49 @@
...
+  "y",
+  "z",
+];
+let arrayId = prompt("Please enter your number");
+let output = document.getElementById("main");
+function printValue() {
+  output.innerHTML = letters[arrayId];
+}
+function getValue() {
+  if (arrayId >= 0 && arrayId <= 25) {
```
Comment (ruback105) [inline Majasdarbs/06-majasdarbs/homework/script.js]: instead of hardcoding 25 you can use array length - 1

Comment (ruback105) [inline Majasdarbs/06-majasdarbs/homework/script.js]: instead of hardcoding 25 you can use array length - 1

**Pull request closed without merging**